### ISSUE TITLE
Status of this plugin

### ISSUE THREAD

**Issue by kuhball** [pretalx/pretalx-socialauth#1]
Hi, thanks for developing pretalx! 

Looking at this plugin it seems to be only the template without any logic, am I mistaken? What was the intention with this plugin? I saw the initial pull request this was created for. I would like to use Oauth with Nextcloud as a provider. 

Looking at Oauth implementations for django I found this https://django-allauth.readthedocs.io/en/latest/overview.html . Does the approach of the tool comply with your intention? If this is the case I would try to create a PR using the django-allauth.

**Comment by rixx** [comment]
You are right, this is not a usable plugin, only a placeholder. People expressed interest in creating pluggable auth in pretalx in the past, so I created this repo for them, but they didn't follow up. If you want to work on it, that would be very cool – just be advised that pretalx currently has no plugin hooks at all to support external authorisation. Part of working on this plugin would be to figure out which plugin hooks would be needed for that – possibly by taking inspiration from the pretix implementation.

**Comment by vmx** [comment]
@kuhball The one wanted to work on it was me. I sadly didn't find the time to do it as proper plugin. I ended up hacking something directly into pretalx that worked for GitHub authentication. Those changes are all in this commit: https://github.com/vmx/pretalx/commit/e25b98a1dd39f16621803434fbb774edd34eb5e5. I post it here as it might help with creating this plugin properly.

IIRC, one big issue was to combine the allauth profiles with pretalx profiles as they work in different ways (sorry, I don't remember the details). Though all this might have changed since I tried.
